src/modules/0.0.23/aws_lambda/entity/lambda_function.ts

```diff
@@ -130,6 +141,15 @@ export class LambdaFunction {
   })
   tags?: { [key: string]: string };
 
+  @Column('varchar', { array: true, nullable: true })
+  subnets?: string[];
```

Comment:
in other tests where we add subnets like this, we also add an `after_install.sql` and `before_uninstall.sql` to add/drop a constraint to make sure all the string inserted as a subnet actually belongs to the subnet table.